### PR TITLE
Web socket code sample

### PR DESCRIPTION
**Pull request by venkanavaneetham**
KC, 

Added code sample for Websocket code sample related to Paho websocket related, can use anyone of the platform e.g. java, python, .net fw, core, nodejs, or go.

https://gdsjira.ship.gov.sg/browse/SENSE-5635

Thanks.

### CODE REVIEW

File: User Guide/Projects.md [hunk 1]
```diff
@@ -718,6 +718,12 @@ To set up **Web socket connection**:
 
 ![Image not Available](/assets/Fig200.png)
 
+## Websocket code sample ## 
+
+Here is the demo code samples related to Paho websocket related, custom business logic is not included. Download the files and use for reference, can use anyone of the platform e.g. java, python, .net fw, core, nodejs, or go.
```
Comment (lohkianchai) [inline User Guide/Projects.md]: Here is the demo code sample with Paho library, custom business logic is not included. Download the file and use for reference.

File: User Guide/Projects.md [hunk 2]
```diff
@@ -718,6 +718,12 @@ To set up **Web socket connection**:
...
+
+- <a href="/assets/golang websocket.zip" download>Click to download</a> golang websocket.zip.
```
Comment (lohkianchai) [inline User Guide/Projects.md]: rename zip file to golang-websocket.zip

File: User Guide/Projects.md [hunk 3]
```diff
@@ -718,6 +718,12 @@ To set up **Web socket connection**:
...
+
 ## Publishing using HTTP REST ##
```
Comment (lohkianchai) [inline User Guide/Projects.md]: Change to
Publish using HTTP REST

File: User Guide/Projects.md (hunk 1 above)
Comment (venkanavaneetham) [reply User Guide/Projects.md]: ok

File: User Guide/Projects.md (hunk 2 above)
Comment (venkanavaneetham) [reply User Guide/Projects.md]: ok

File: User Guide/Projects.md (hunk 3 above)
Comment (venkanavaneetham) [reply User Guide/Projects.md]: ok

**Review by lohkianchai — APPROVED** [review]
ok

**Pull request merged**